backend/src/modules/tournament-tree/core/domain/FightingTeamsGroup.ts

```diff
@@ -0,0 +1,45 @@
+import { FightingTeamsGroupId } from './FightingTeamsGroupId';
+import { TournamentTeam } from './TournamentTeam';
+
+export class FightingTeamsGroup {
+  readonly fightingTeamsGroupId: FightingTeamsGroupId;
+  firstTeam: TournamentTeam | undefined;
+  secondTeam: TournamentTeam | undefined;
+  readonly fightingTeamsGroupLevel: number;
+  nextMatchId: FightingTeamsGroupId | undefined;
+
+  private constructor(props: {
```

Comment:
prywatny konstruktor? Nigdzie nie będziesz musiał tworzyć instancji tej klasy? To jest obiekt domenowy to chyba gdzieś zajdzie potrzeba stworzenia sobie obiektów typu FightingTeamsGroup?